### PR TITLE
Add business

### PR DESCRIPTION
**Pull request by stevenc987**
*Issue #:*

*Description of changes:*

*API Code Review Checklist:*
 - #### Common
    - [ ] Proper naming conventions followed on variables, classes, definitions etc.
    - [ ] Proper logging levels are used in the code 
 - #### REST 
    - [ ] Endpoints follows agreed pattern and organized around resources
    - [ ] Conform to http semantics
    - [ ] Proper filtering and pagination is provided for large amounts of data
 - #### Code structure
    - [ ] Resources is used for request payload processing, validations, response codes, messages etc.
    - [ ] Service is used for business logic, 3rd party communication management. Maintains proper folder structure.
    - [ ] Model is used for database models.
    - [ ] Schemas is used for json schema
    - [ ] Util contains commonly used utility functions, constants
 - #### Test Cases
    - [ ] Unit test cases with happy and sad path for service, model (?) layers
    - [ ] Endpoints test cases with happy and sad path for resources
    - [ ] Mocking of services (mainly 3rd party dependencies) provided if applicable
    - [ ] Postman collection tests provided for endpoints with happy path
 - #### Code Quality
    - [ ] Proper code coverage for all the layers (percentage 80, 90 ?)
    - [ ] Linting
- #### DB Migrations
    - [ ] Alembic is used to manage database migration. [Reference](https://github.com/bcgov/namex/blob/master/docs/database.md)

*UI Code Review Checklist:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: auth-web/public/config/configuration.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
   "VUE_APP_COPS_REDIRECT_URL": "http://localhost:8081",
   "VUE_APP_PAY_ROOT_API": "https://pay-api-dev.pathfinder.gov.bc.ca/api/v1",
-  "VUE_APP_AUTH_ROOT_API": "https://auth-api-post-dev.pathfinder.gov.bc.ca/api/v1"
+  "VUE_APP_AUTH_ROOT_API": "http://localhost:5000/api/v1"
```
Comment (sutherlanda) [inline auth-web/public/config/configuration.json]: Can you revert this change?

File: auth-web/src/router.ts [hunk 2]
```diff
@@ -37,6 +38,7 @@ const routes = [
   { path: '/returnpayment/:paymentId/transaction/:transactionId', component: PaymentReturnForm, props: mapReturnPayVars, meta: { requiresAuth: true } },
   { path: '/createaccount', component: CreateAccount, meta: { requiresAuth: false } },
   { path: '/dashboard', component: Dashboard, meta: { requiresAuth: true } },
+  { path: '/addbusiness', component: AddBusinessForm },
```
Comment (sutherlanda) [inline auth-web/src/router.ts]: You'll need the `requiresAuth` meta property here.  See the other routes for an example.

File: auth-web/src/components/auth/AddBusinessForm.vue [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+    return (this.$refs.form as Vue & { validate: () => boolean }).validate()
+  }
+
+  private getIFrameContent (): Window {
+    return (this.$refs.iframeContent as Vue & { contentWindow: Window }).contentWindow
+  }
+
+  private redirectToNext (): void {
+    // transition to business contact UI
+    setTimeout(() => {
```
Comment (pksumesh) [inline auth-web/src/components/auth/AddBusinessForm.vue]: We don't need this timeout

File: auth-web/src/components/auth/AddBusinessForm.vue [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+  businessStore = getModule(BusinessModule, this.$store)
+
+  businessNumber: string = ''
+  passcode: string = ''
+
+  private isFormValid (): boolean {
+    return (this.$refs.form as Vue & { validate: () => boolean }).validate()
+  }
+
+  private getIFrameContent (): Window {
```
Comment (pksumesh) [inline auth-web/src/components/auth/AddBusinessForm.vue]: This function is not needed

File: auth-web/src/components/auth/AddBusinessForm.vue [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+import { Component, Prop } from 'vue-property-decorator'
+import { getModule } from 'vuex-module-decorators'
+import BusinessModule from '../../store/modules/business'
+import configHelper from '../../util/config-helper'
+import iframeServices from '../../services/iframe.services'
+
+@Component
+export default class AddBusinessForm extends Vue {
+  showPasscode = false
+  loginError = ''
```
Comment (pksumesh) [inline auth-web/src/components/auth/AddBusinessForm.vue]: can we have a better naming than loginError
